react/react1/week2/todo-list/src/components/Timer.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from "react";
+import { useState } from "react";
+
+export const Timer = () => {
+  const [count, setCount] = useState(() => {
```

Comment:
Don't see why this has to be defined as a function. It's not just a counter? Couldn't it be
`useState(0)`

Reply:
Hi :) thanks for the feedback - right, that's simply a counter, so in this case it can be just useState(0). Fixed it 🤓